boltz_client/cli.py

```diff
@@ -98,11 +98,21 @@ def refund_swap(
 
 @click.command()
 @click.argument("sats", type=int)
-def create_reverse_swap(sats: int):
+@click.argument("direction", type=str, default="send")
+def create_reverse_swap(sats: int, direction: str):
     """
     create a reverse swap
     """
     client = BoltzClient(config)
+
+    if direction == "receive":
+        sats = client.add_reverse_swap_fees(sats)
+    elif direction == "send":
```

Comment:
How about an enum for those `receive` and `send`?

Reply:
f346c90